src/submitExtrinsic.ts

```diff
@@ -38,28 +38,32 @@ export function getSignerAddress(signer: KeyPairSigner | GenericSigner) {
   }
 }
 
-export async function submitExtrinsic(
+export async function submitAndSignExtrinsic(
```

Comment:
Should we extract the signing part out of this function? So we have `submitExtrinsic()`, `signExtrinsic()` and `submitAndSignExtrinsic()` which then calls both.

Reply:
Done. I also renamed to `submitAndSignExtrinsic` to `signAndSubmitExtrinsic`, which makes more sense.